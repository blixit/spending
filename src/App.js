import React from 'react';

import ThemeProvider from 'ui/theming/theme';
import { EventProvider } from 'core/events/provider';

import RoutedApp from './components/menu/RoutedApp';

import './App.css';

function App() {
  return (
    <ThemeProvider>
      <EventProvider>
        <RoutedApp />
      </EventProvider>
    </ThemeProvider>
  );
}

export default App;
