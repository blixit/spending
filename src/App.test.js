import React from 'react';

import render from 'testing/render';

import App from './App';

describe('[App]', () => {
  it('renders properly', () => {
    const app = render(<App />);
    expect(app.toJSON()).toMatchSnapshot();
  });
});
