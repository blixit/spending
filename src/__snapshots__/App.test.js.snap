// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[App] renders properly 1`] = `
Array [
  <div
    className="sc-bdVaJa kjZkLq"
  >
    <div
      className="sc-bwzfXH gouoTZ"
    />
  </div>,
  <div
    className="sc-iRbamj dOQpTu"
  >
    <a
      className="sc-jlyJG epKrOG"
      href="/"
      iscurrentpage="true"
      onClick={[Function]}
    >
      Dépenses
    </a>
    <a
      className="sc-jlyJG kfIYRT"
      href="/new"
      iscurrentpage="false"
      onClick={[Function]}
    >
      Nouvelle dépense
    </a>
    <a
      className="sc-jlyJG kfIYRT"
      href="/statistiques"
      iscurrentpage="false"
      onClick={[Function]}
    >
      Statistiques
    </a>
    <a
      className="sc-jlyJG kfIYRT"
      href="/admin"
      iscurrentpage="false"
      onClick={[Function]}
    >
      Admin
    </a>
  </div>,
]
`;
